platform/darwin/src/MGLStyle.mm

```diff
@@ -254,5 +257,21 @@ - (void)removeStyleClass:(NSString *)styleClass
     }
 }
 
+- (void)setImage:(MGLImage *)image forName:(NSString *)name
+{
+    if (image && name)
```

Comment:
By default, passing in `nil` for a `nonnull` parameter is only a warning rather than an error, so `NSAssert()` is for the benefit of developers who ignore their warnings. (Tsk, tsk!) On the other hand, we haven’t been disciplined about asserting in similar cases – `-addStyleClass:` and `-removeStyleClass:` are more examples of this – so it’s a bit unfair of me to nitpick about this detail. 😊